### PR TITLE
Added StyleCI to CI/CD list

### PR DESCRIPTION
**Pull request by jbrooksuk**
https://styleci.io

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,7 @@ Table of Contents
   * http://ftploy.com/ - 1 project w/ unlimited deployments
   * https://deployhq.com/ - 1 project w/ 10 daily deployments
   * https://hub.jazz.net/ - 60 minutes of free build time / month.
+  * https://stylecio.io/ - Public GitHub repositories only.
```
Comment (ripienaar) [inline README.md]: extra 'o' here

Comment (jbrooksuk) [inline README.md]: Doh.

**Comment by jbrooksuk** [comment]
Fixed.

**Comment by ripienaar** [comment]
thank you :)

**Comment by jbrooksuk** [comment]
Thanks! CC @GrahamCampbell

**Comment by GrahamCampbell** [comment]
@jbrooksuk https://github.com/ripienaar/free-for-dev/pull/175, https://github.com/ripienaar/free-for-dev/pull/170

**Comment by ripienaar** [comment]
yeah was added before but PR wouldnt merge cleanly, first clean PR wins unfort :)

**Comment by jbrooksuk** [comment]
We all wanted the same thing :)

**Comment by GrahamCampbell** [comment]
yeh :)

**Comment by GrahamCampbell** [comment]
Wait, @jbrooksuk you added this into the wrong section...

**Comment by GrahamCampbell** [comment]
This should be in the "code quality" section shouldn't it?

**Comment by jbrooksuk** [comment]
Probs,

**Pull request merged**